spec/app/models/mdm/host_spec.rb

```diff
@@ -1,4 +1,6 @@
 RSpec.describe Mdm::Host, type: :model do
+  include_context 'Rex::Text'
```

Comment:
Good refactoring, but in general, I'd like people to separate refactoring PRs, like the change in this file from bug fix or feature adding PRs as refactor should not affect behavior and it can also help with cherry-picking if another branch wants just the refactor or just needs the bug fix.